source/jormungandr/jormungandr/scenarios/tests/qualifier_tests.py

```diff
@@ -242,3 +243,57 @@ def qualifier_nontransport_duration_with_tc_test():
     journey.sections[-1].duration = 864
 
     assert qualifier.get_nontransport_duration(journey) == 2797
+
+
+def robust_journey_test():
+    journey = response_pb2.Journey()
+    assert is_robust_journey(journey) == False
+
+    journey.sections.add()
+    journey.sections[0].type = response_pb2.PUBLIC_TRANSPORT
+    journey.sections[0].uris.physical_mode = "physical_mode:Bus"
+
+    assert is_robust_journey(journey) == False
+
+    journey.sections[0].uris.physical_mode = "physical_mode:Train"
+
+    assert is_robust_journey(journey) == True
+
+    journey.sections.add()
+    journey.sections[1].type = response_pb2.PUBLIC_TRANSPORT
+    journey.sections[1].uris.physical_mode = "physical_mode:Bus"
+
+    assert is_robust_journey(journey) == False
+
+    journey.sections[1].uris.physical_mode = "physical_mode:Train"
+    assert is_robust_journey(journey) == True
+
+
+def robust_journey_with_fallbacks_test():
+    journey = response_pb2.Journey()
+    assert is_robust_journey(journey) == False
+
+    journey.sections.add()
+    journey.sections[0].street_network.mode = response_pb2.Bike
+
+    # no public transport in the journey, so it should not be tagged as robust
+    assert is_robust_journey(journey) == False
+
+    journey.sections.add()
+    journey.sections[1].type = response_pb2.PUBLIC_TRANSPORT
+    journey.sections[1].uris.physical_mode = "physical_mode:Bus"
+
+    assert is_robust_journey(journey) == False
```

Comment:
Is this test useful considering you already test physical modes in the previous test `robust_journey_test`?